sdk/communication/communication-jobrouter/samples-dev/ClassificationPolicyCrud.ts

```diff
@@ -0,0 +1,210 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary Classification policy crud
+ */
+import { DistributionPolicy, ExceptionPolicy, JobQueue, QueueLengthExceptionTrigger, RouterAdministrationClient } from "../src";
+
+// Load the .env file (you will need to set these environment variables)
+import * as dotenv from "dotenv";
+import { ClassificationPolicy } from "../src";
+import { ClassificationPolicyItem } from "../src";
+import { assert } from "chai";
+dotenv.config();
+
+const connectionString = process.env["COMMUNICATION_CONNECTION_STRING"] || "";
+
+
+// Create an classification policy
+const createClassificationPolicy = async (): Promise<void> => {
```

Comment:
For readability, I recommend defining these as good old functions.

```ts
function createClassificationPolicy(): Promise<void> { 
```

Reply:
Choose to keep the modern signature here.

Reply:
Since these samples are the entry point for our customers, I strongly suggest favoring readability with

```ts
function createClassificationPolicy(): Promise<void> { 
```